Cargo.toml

```diff
@@ -24,7 +24,7 @@ petgraph = "0.4"
 rustsec = { version = "0.15.2", features = ["dependency-tree"] }
 serde = { version = "1", features = ["serde_derive"] }
 serde_json = "1"
-
+cargo-edit = { git = "https://github.com/killercup/cargo-edit#cargo-upgrade" }
```

Comment:
It looks like Cargo doesn't like this dependency?

https://github.com/RustSec/cargo-audit/pull/151/checks?check_run_id=255223348

I think you can probably just do:

```
cargo-edit = { git = "https://github.com/killercup/cargo-edit.git" }
```